packages/ember-glimmer/lib/environment.js

```diff
@@ -224,12 +227,15 @@ export default class Environment extends GlimmerEnvironment {
     return false;
   }
 
-  getComponentDefinition(path) {
+  getComponentDefinition(path, parentMeta = {}) {
```

Comment:
@krisselden what would be the right way to get the parent meta info in the dynamic case?

So [here](https://github.com/emberjs/ember.js/blob/c8824b01ccc1105fc1e9191b23ac905efd760cd4/packages/ember-glimmer/lib/syntax/dynamic-component.js#L49) and [here](https://github.com/emberjs/ember.js/blob/c8824b01ccc1105fc1e9191b23ac905efd760cd4/packages/ember-glimmer/lib/syntax/dynamic-component.js#L12).